util/xs_scripts/rvv_vlen/test_matrix.py

```
@@ -0,0 +1,188 @@
+        help="XS config script used when emitting run commands",
+    )
+    parser.add_argument(
+        "--emit-commands",
+        action="store_true",
+        help="Print suggested gem5 command lines for available local binaries",
+    )
+    args = parser.parse_args(argv)
+
+    matrix = build_matrix()
+    print(f"# XS-GEM5 RVV VLEN matrix: {len(matrix)} cases "
+          f"({len(UPSTREAM_RESOURCES)} resources × {len(XS_VLENS)} VLENs)")
```

Ruff (0.16.0)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)